### DISCUSSION TITLE
Support native TOML support following tox-dev/tox#3353

### DISCUSSION THREAD

**Discussion by paddyroddy** [ymyzk/tox-gh-actions#197] [Ideas]
`tox` now supports full support TOML support. However, for this plugin to work, one still needs to use the `legacy_ini_support`. Is it possible to adapt this new format?

**Comment by ymyzk** [comment]
Thanks for opening a discussion. I didn't know that tox now supports TOML natively. It's a good idea to implement it for tox-gh-actions as well. I'll look into more details shortly.
